openedx/adg/lms/webinars/helpers.py

```diff
@@ -288,3 +299,81 @@ def has_any_field_changed(item, old_item_data, fields_to_check):
         bool: True if any field has been changed, False otherwise
     """
     return any([getattr(item, field) != old_item_data.get(field) for field in fields_to_check])
+
+
+def update_all_webinar_registrations_to_new_email(old_email, new_email):
+    """
+    Uses the `WebinarRegistrationEmailUpdateForm` form to update all
+    the webinar registrations associated with the old email
+
+    Args:
+        old_email (str): email to fetch all the registrations
+        new_email (str) email to update all the associated registrations
+    """
+    from openedx.adg.lms.webinars.forms import WebinarRegistrationEmailUpdateForm
+
+    registration_email_information = {
+        'old_email': old_email,
+        'new_email': new_email
+    }
+
+    form = WebinarRegistrationEmailUpdateForm(data=registration_email_information)
+    form.save()
+
+
+def get_arabic_or_english_datetime(date_time_object, output_format):
+    """
+    Get arabic or english datetime formatted string based the user selected language
+
+    Args:
+        date_time_object (datetime): Datetime date, time or datetime object
+        output_format (str): Format of the datetime object
+
+    Returns:
+        str: Formatted string for date
+    """
+    if is_user_language_arabic():
+        return construct_arabic_datetime_string(date_time_object, output_format)
+
+    return convert_date_time_zone_and_format(date_time_object, WEBINAR_DEFAULT_TIME_ZONE, output_format)
+
+
+def construct_arabic_datetime_string(date_time_object, output_format):
+    """
+    Create the correct Arabic language localized date string based on the string
+    format provided
+
+    Args:
+        date_time_object (datetime): Datetime date, time or datetime object
+        output_format (str): Output formatting for the datetime object
+
+    Returns:
+        str: Localized arabic output string in the specified format
+    """
+    date_in_AST = convert_timezone(date_time_object, WEBINAR_DEFAULT_TIME_ZONE)
+
+    with override('ar'):
+        if output_format == WEBINAR_DATE_TIME_FORMAT:
+            return _('{week_day}, {month} {day}, {year} {hours}:{minutes} {time_format} AST').format(
```

Comment:
Do we really need to do _() over the formatting string?

Reply:
Removed these function calls.